### ISSUE TITLE
Pinball Dreams and Fantasies (21st Century) crashes vAmiga

### ISSUE THREAD

**Issue by Gianmarco72** [dirkwhoffmann/vAmiga#344]
When the first ball is ready to be played vAmiga crash and quit with an unexpected error on macOS 😱:
<img width="912" alt="Schermata 2020-05-18 alle 15 56 47" src="https://user-images.githubusercontent.com/45571833/82223420-ef3e5980-9922-11ea-9143-6e4969c900b0.png">
It happens with any table (also with Pinball Fantasies).

Works on UAE 👍🏻:
<img width="1072" alt="Schermata 2020-05-18 alle 16 00 56" src="https://user-images.githubusercontent.com/45571833/82223608-26ad0600-9923-11ea-9e9f-0a16b1dbb1cc.png">

**Comment by dirkwhoffmann** [comment]
With the latest code changes, it doesn't seem to crash any more:

<img width="923" alt="Bildschirmfoto 2020-05-19 um 13 13 40" src="https://user-images.githubusercontent.com/12561945/82320105-06d51b00-99d3-11ea-93fd-4e8eb695533a.png">

There was also a  keyboard issue w.r.t. handshaking. Here is what the HRM says: 

 ```
   // Handshake detection logic

    /* "The handshake is issued by the processor pulsing the SP line low for a
     *  minimum of 75 microseconds." [HRM 2nd edition]
     *
     * "This handshake is issued by the processor pulsing the SP line low then
     *  high. While some keyboards can detect a 1 microsecond handshake pulse,
     *  the pulse must be at least 85 microseconds for operation with all
     *  models of Amiga keyboards." [HRM 3rd editon]
     */
    int diff = (spHigh - spLow) / 28;
    bool accept = diff >= 45;

    if (accept) {

        debug(KBD_DEBUG, "Accepting handshake (SP low for %d usec)\n", diff);
        processHandshake();
    }
```

Kickstart Rom pulses the SP line according to the specs, so I gave vAmiga a keyboard that recognizes a handshake when the SP line is low for at least 45 microseconds. But this game (and maybe others, too) don't comply to the specs. Here, the handshake pulses last for 15 microseconds, only. 

To solve this, I decided to ship the next version of vAmiga with a luxury high-speed handshake detection keyboard. Fortunately, it's not so difficult. Line

      bool accept = diff >= 45;

needs to be changed to this:

      bool accept = diff >= 1;

Now, the keyboard works like a charm in this game. 😎

**Comment by dirkwhoffmann** [comment]
Fixed in v0.9.7
